docs/user/grid.md

```diff
@@ -0,0 +1,106 @@
+Everyone is permitted to copy and distribute verbatim copies of this license 
+document, but changing it is not allowed.
+
+This version of the GNU Lesser General Public License incorporates the terms
+and conditions of version 3 of the GNU General Public License, supplemented by
+the additional permissions listed below.
+-->
+
+# Grid plots
+---
+
+Grid plots are a way to visualise data from a GRID file of a SuperDARN radar(s). Please read RST documentation on how to process [GRID files and GRD files](https://radar-software-toolkit-rst.readthedocs.io/en/latest/user_guide/make_grid/) from grid files.    
+
+Gridded given [parameter](https://radar-software-toolkit-rst.readthedocs.io/en/latest/references/general/grid/) such as:
+
+| Name                    | parameter name |
+| ----------------------- | -------------- |
+| Light-of-sight Velocity | `vec`          |
+| Power                   | `pwr`          |
+| Spectral Width          | `wdt`          |
+
+and a time or record number projected onto a polar format plot in [AACGMv2](http://superdarn.thayer.dartmouth.edu/aacgm.html) coordinates. 
+
+
+Currently, grid plots in pyDARN get the geomagnetic positions from the GRID file [`mlon` and `mlat`](https://pypi.org/project/aacgmv2/) that uses AACGMv2 coordinates. In the future, pyDARN will generate the geographical position, which will bring support for not standard gridded vector layouts.
+
+### Basic usage
+
+pyDARN and pyplot need to be imported, as well as any GRID file needs to be [read in](https://pydarn.readthedocs.io/en/master/user/SDarnRead/):
+
+```python
+import matplotlib.pyplot as plt
+import pydarn
+
+#Read in grid file using SuperDARDRead, then read_grid
+file = "path/to/grid/file"
+SDarn_read = pydarn.SuperDARNRead(file)
+grid_data = SDarn_read.read_grid()
+
+```
+With the grid data loaded as a list of dictionaries (`grid_data` variable in above example), you may now call the `plot_grid` method. Make sure you tell it which time or record (numbered from first recorded in file, counting from 0):
+```python
+import datetime as dt
+
+stime = dt.datetime(year, month, day, hour, minute)
+gridplot = pydarn.Grid.plot_grid(grid_data, start_time=stime)
+plt.show()
+
+```
+In this example, the record at `stime` was plotted with the defaulted parameter being gridded line-of-sight velocity:
+![](../imgs/grid_1.png)
+
+You might have noticed that the variable `gridplot` in the examples above actually holds some information. This contains the AACGM latitude and longitude of the gridded vectors plotted, as well as the data. If you instead change `gridplot` to 3 separate variables, it will return the latitude, longitude, and data info into separate variables:
+```python
+lats,lons,data=pydarn.Grid.plot_grid(grid_data, start_time=stime)
+```
+
+### Additional parameters
+
+In addition to gridded  line-of-sight velocity, you can choose one of three other data products to plot by setting `parameter=String name`:
```

Comment:
```suggestion
In addition to the gridded line-of-sight velocity, you can choose one of two other data products to plot by setting `parameter=String name`:
```
This is the same table from the top of the page too, I think we need this one, but maybe the one at the top isn't required. I'll see how it reads on read the docs.

Reply:
okay remove the one you don't prefer in the next review